bokeh/charts/builder/bar_builder.py

```diff
@@ -37,7 +37,40 @@
 
 def Bar(values, cat=None, stacked=False, xscale="categorical", yscale="linear",
         xgrid=False, ygrid=True, continuous_range=None, **kw):
+    """ Create a Bar chart using the `BarBuilder` to
+    render the geometry from `values`, `cat` and `stacked`.
 
+    Args:
+        values (iterable): iterable 2d representing the data series
+            values matrix.
+        cat (list or bool, optional): list of string representing the categories.
+            (Defaults to None)
+        stacked (bool, optional): to see the bars stacked or grouped.
+            (Defaults to False, so grouping is assumed)
+        continuous_range(Range1d, optional): Custom continuous_range to be
+            used. (Defaults to None)
+
+    In addition the the parameters specific to this chart,
+    :ref:`charts_generic_arguments` are also accepted as keyword parameters.
+
```

Comment:
To be consistent with other docstrings, there should  be an `Examples:` lead in (and in all the other functions as well)